internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/pubsub/manager/TopicManagerE2ETest.java

```diff
@@ -270,7 +270,7 @@ public void testMetadataApisForNonExistentTopics() throws ExecutionException, In
     assertEquals(topicManager.getLatestOffsetCached(nonExistentTopic, 1), StatsErrorCode.LAG_MEASUREMENT_FAILURE.code);
   }
 
-  @Test(timeOut = 3 * Time.MS_PER_MINUTE)
+  @Test(timeOut = 3 * Time.MS_PER_MINUTE, invocationCount = 50)
```

Comment:
This is good to test that the flakiness is fixed, but I guess we'll revert it before merging, right?

Reply:
Yeah once verified, I'll remove it.

Reply:
All 50 iterations passed. (+100 on internal CI).